test/unit/cmds/merge_cmds/show.test.ts

```diff
@@ -0,0 +1,382 @@
+        }
+      },
+      data: {
+        displayField: null,
+        name: 'ToBeDeletedType',
+        description: '',
+        fields: [
+          {
+            id: 'number',
+            name: 'Number',
+            type: 'Integer',
+            localized: false,
+            required: false,
+            validations: [],
+            disabled: false,
+            omitted: false
+          }
+        ]
+      }
+    },
+    {
+      changeType: 'delete',
+      entity: {
+        sys: {
+          id: 'newType',
+          linkType: 'ContentType',
+          type: 'Link'
+        }
+      }
+    },
+    {
+      changeType: 'update',
+      entity: {
+        sys: {
+          id: 'toBeModifiedType',
+          linkType: 'ContentType',
+          type: 'Link'
+        }
+      },
+      patch: [
+        {
+          op: 'remove',
+          path: '/fields/1'
+        },
+        {
+          op: 'replace',
+          path: '/fields/0/name',
+          value: 'yes'
+        },
+        {
+          op: 'add',
+          path: '/fields/1',
+          value: {
+            id: 'numberEditted',
+            name: 'number-editted',
+            type: 'Integer',
+            localized: false,
+            required: false,
+            validations: [],
+            disabled: false,
+            omitted: false
+          }
+        }
+      ]
+    }
+  ]
+}
+
+describe('merge show command', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('stops early if both env have the app installed', async () => {
+    const appInstalledInBothEnvs =
+      await appInstallUtils.checkAndInstallAppInEnvironments(
+        mockedClient,
+        'space',
+        ['source', 'target'],
+        'app-id',
+        false
+      )
+
+    expect(appInstalledInBothEnvs).toBe(true)
+    expect(mockedClient.appInstallation.get).toHaveBeenCalledTimes(2)
+  })
+
+  it('installs app to both envs if none of them have it installed', async () => {
+    mockedClient.appInstallation.get = jest.fn().mockImplementationOnce(() => {
+      throw { name: 'NotFound' }
+    })
+
+    const appInstalledInBothEnvs =
+      await appInstallUtils.checkAndInstallAppInEnvironments(
+        mockedClient,
+        'space',
+        ['source', 'target'],
+        'app-id',
+        true
+      )
+
+    expect(appInstalledInBothEnvs).toBe(true)
+    expect(mockedClient.appInstallation.get).toHaveBeenCalledTimes(2)
+    expect(mockedClient.raw.put).toHaveBeenCalledTimes(2)
+  })
+
+  it('installs the app in the other env if only one has it installed', async () => {
+    const appInstalledInBothEnvs =
+      await appInstallUtils.checkAndInstallAppInEnvironments(
+        mockedClient,
+        'space',
+        ['source', 'target'],
+        'app-id',
+        true
+      )
+
+    expect(appInstalledInBothEnvs).toBe(true)
+    expect(mockedClient.appInstallation.get).toHaveBeenCalledTimes(2)
+  })
+
+  it('call the create changeset actions', async () => {
```

Comment:
```suggestion
  it('calls the create changeset actions', async () => {
```
Sorry, I messed up my suggestion the first time around.